src/Theme_Command.php

```diff
@@ -211,8 +213,11 @@ protected function get_all_items() {
 	 *
 	 *     $ wp theme activate twentysixteen
 	 *     Success: Switched to 'Twenty Sixteen' theme.
+	 *
+	 * @param array $args
+	 * @param array $assoc_args
 	 */
-	public function activate( $args = array() ) {
+	public function activate( $args, $assoc_args = [] ) {
```

Comment:
This changed `$args` from optional to required. Was this intended?

Reply:
Yes, because it is actually required as per the synopsys, and WP-CLI always passes both arguments to commands anyway. I've never seen `$args` or `$assoc_args` be null actually, so should even change this to `( $args, $assoc_args )`.